docs/src/reference/asciidoc/core/intro/requirements.adoc

```diff
@@ -71,8 +71,6 @@ As a guide, the table below lists the Hadoop-based distributions against with th
 | Apache Hadoop     | 2.6.x
 | Apache Hadoop     | 2.4.x
 | Apache Hadoop     | 2.2.x
-| Apache Hadoop     | 1.2.x
-| Apache Hadoop     | 1.1.x
```

Comment:
Did we check to see if any of these other distros bundle Hadoop 1.x? I was originally thinking we can remove this list as a whole.